### DISCUSSION TITLE
question for files dot cat

### DISCUSSION THREAD

**Discussion by vay3t** [Stellarium/stellarium#730] [Q&A] [answered]
i need to process information of galaxies of stellarium, and i think to use python for this, but the problem is that the catalog is a "binary", how to parse this information or convert to json/csv?

**Comment by alex-w** [comment, marked as answer]
It's maybe strange, but any .cat file doesn't contains data for galaxies. Please read the Stellarium User Guide to understand formats of our files.

Hint: https://github.com/Stellarium/stellarium/blob/master/nebulae/default/catalog.txt
